### ISSUE TITLE
Pretrained models for quick test

### ISSUE THREAD

**Issue by HaozheQi** [JacobChalk/TIM#1]
Dear Jacob,

Thank you very much for sharing this amazing work. I am trying to reproduce the vision-based action recognition result on EPIC Kitchen dataset. However, I found it takes really long time to extract the visual features. Therefore, I wonder if you could either share the pre-trained VideoMAE model or share the extracted features. That would be really helpful!

Meanwhile, I found there is only a visual+audio model shared [here](https://github.com/JacobChalk/TIM/tree/main/recognition#pretrained-models). I wonder if it is possible to also share the visual-only model. 

And my last question is if you have tried with only the `omnivore` feature without the VideoMAE feature. Would the performance drop a lot?

Sorry for my tons of questions. Thank you in advance for your help!

**Comment by JacobChalk** [comment]
Hi,

Unfortunately we are not able to share the extracted features due to the size required, but we have shared our pretrained model for VideoMAE [here](https://www.dropbox.com/scl/fi/kr14exj9ipfcoth01thld/videomae_epic.pth.tar?rlkey=d9jqjqp2b3zy1440qcbdog3x4&dl=0). You can find this in the`feature_extractors/VideoMAE/ReadME.md` also. 

I also just uploaded the visual only model [here](https://www.dropbox.com/scl/fi/udqes9jcyl1at05xljdgn/epic_visual_only.pth.tar?rlkey=56pe4jjnlyjxg1o2tris4npyi&st=ih7fyqvp&dl=0).

For your final question, we did try omnivore only, but this did suffer a bit of a performance drop, I can't remember the exact numbers but in particular the verb metric was worse by about 1-2%.
